cmdstanpy/model.py

```diff
@@ -261,7 +261,7 @@ def optimize(
                 msg = '{} Got returned error code {}'.format(
                     msg, stanfit._retcode(dummy_chain_id)
                 )
-            raise Exception(msg)
+            print(msg)
```

Comment:
I don't think just printing the message is sufficient - if the retcode is not 0, the csv files are junk and should be ignored - don't parse them.

Reply:
I've tested it out and in case of fbprophet when the exit code is 70 the csv file is still readable (just the content is not to be trusted). After all we check the csv content separately.
I will push the change where we raise exception if the exit code is != 0 and != 70

Reply:
sounds good!